fighthealthinsurance/common_view_logic.py

```
@@ -1812,7 +1814,8 @@ async def format_response(response: dict[str, str]) -> str:
             pubmed_context=pubmed_context,
             ml_citations_context=ml_citation_context,
         )
-        filtered_appeals: Iterator[str] = filter(lambda x: x != None, appeals)
+         # Only filters out None
+        filtered_appeals: Iterator[str] = filter(lambda x: x != None, appeals) 
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)